appendix/matching/main.tex

```diff
@@ -0,0 +1,503 @@
+\chapter{An introduction to matching games}
+\label{app:matching}
+
+\graphicspath{{appendix/matching/paper/img/}}
+
+Matching games form a part of game theory that were formally introduced by Gale
+and Shapley in their seminal work~\cite{Gale1962}. These games allow for the
+allocation of resources and partnerships in a mathematically fair way.
+Typically, a matching game is defined by two sets of players (called parties)
+that each have preferences over at least some of the elements of the other set.
+The objective of the game is then to find a mapping between the sets of players
+in which everyone is \emph{happy enough} with their match(es).
+
+This appendix does not contain any novel mathematics, but it does offer an
+introduction to matching games and their variants. Studying this branch of
+mathematics has contributed to a significant amount of the research conducted
+for this thesis, hence its inclusion here. That research has culminated in the
+development of a Python library for solving various matching games, \matching.
+Among other uses, the \matching\ library proves instrumental in the practical
+implementation of the novel method described in Chapter~\ref{chp:kmodes}.
+
+The \matching\ library has been developed as a research tool and adheres to the
+best practices discussed in Chapter~\ref{chp:intro}. The current version of
+Matching has also been archived on Zenodo under~\doi{10.5281/zenodo.3931026}.
```

Comment:
Do you want to say `current` or should it be more pointing at the version used in this thesis? (Or did you use two different versions -- biosci versus initialisation?)

Reply:
The HR and SA implementations are the same in each case, but I think they use slightly different versions of `matching`. I believe that link goes to the `matching` Zenodo page (which redirects to the most recent version)

I will point at the correct versions in the k-modes chapter and biosci appendix, though

Reply:
:+1:

Reply:
✅ 1e6978c